lib/events/EventManager.ts

```diff
@@ -126,17 +126,20 @@ export default class EventManager {
    *
    * @param event
    */
-  public async update(event: Ensure<CalendarEvent, "uid">): Promise<CreateUpdateResult> {
-    let evt = EventManager.processLocation(event);
-
-    if (!evt.uid) {
-      throw new Error("You called eventManager.update without an `uid`. This should never happen.");
+  public async update(
+    event: Ensure<CalendarEvent, "language">,
+    rescheduleUid: string
```

Comment:
What's the reasoning for reintroducing an additional parameter instead of using the event.uid? @mihaic195